Failed to compile on Win7-64 with Rust 1.38

I would really love to use your wmi module. Unfortunately though, it won't build for me:
```
R:\rs\wmitest>cargo run --release --
   Compiling wmi v0.4.5
error[E0554]: `#![feature]` may not be used on the stable release channel
   --> C:\Users\mark\.cargo\registry\src\github.com-1ecc6299db9ec823\wmi-0.4.5\src\lib.rs:101:1
    |
101 | #![feature(ptr_internals, custom_attribute)]
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error

For more information about this error, try `rustc --explain E0554`.
error: Could not compile `wmi`.

To learn more, run the command again with --verbose.
R:\rs\wmitest>rustc -V
rustc 1.38.0 (625451e37 2019-09-23)
```

--- a/src/connection.rs
+++ b/src/connection.rs
@@ -2,7 +2,7 @@ use crate::utils::check_hres;
 use failure::Error;
 use log::debug;
 use std::ptr;
-use std::ptr::Unique;
+use std::ptr::NonNull;
 use std::rc::Rc;
 use widestring::WideCString;
 use winapi::{
@@ -19,7 +19,7 @@ use winapi::{
             CoCreateInstance, CoInitializeEx, CoInitializeSecurity, CoSetProxyBlanket,
             CoUninitialize,
         },
-        objbase::{COINIT_MULTITHREADED, COINIT_APARTMENTTHREADED},
+        objbase::{COINIT_APARTMENTTHREADED, COINIT_MULTITHREADED},
         objidl::EOAC_NONE,
         wbemcli::{CLSID_WbemLocator, IID_IWbemLocator, IWbemLocator, IWbemServices},
     },
@@ -81,8 +81,8 @@ impl Drop for COMLibrary {
 
 pub struct WMIConnection {
     com_con: Rc<COMLibrary>,
-    p_loc: Option<Unique<IWbemLocator>>,
-    p_svc: Option<Unique<IWbemServices>>,
+    p_loc: Option<NonNull<IWbemLocator>>,
+    p_svc: Option<NonNull<IWbemServices>>,
 }
 
 /// A connection to the local WMI provider, which provides querying capabilities.
@@ -105,7 +105,10 @@ impl WMIConnection {
     /// # Ok(())
     /// # }
     /// ```
-    pub fn with_namespace_path(namespace_path: &str, com_lib: Rc<COMLibrary>) -> Result<Self, Error> {
+    pub fn with_namespace_path(
+        namespace_path: &str,
+        com_lib: Rc<COMLibrary>,
+    ) -> Result<Self, Error> {
         let mut instance = Self {
             com_con: com_lib,
             p_loc: None,
@@ -144,7 +147,7 @@ impl WMIConnection {
             ))?;
         }
 
-        self.p_loc = Unique::new(p_loc as *mut IWbemLocator);
+        self.p_loc = NonNull::new(p_loc as *mut IWbemLocator);
 
         debug!("Got locator {:?}", self.p_loc);
 
@@ -171,7 +174,7 @@ impl WMIConnection {
             ))?;
         }
 
-        self.p_svc = Unique::new(p_svc as *mut IWbemServices);
+        self.p_svc = NonNull::new(p_svc as *mut IWbemServices);
 
         debug!("Got service {:?}", self.p_svc);
 

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -98,7 +98,6 @@
 //!
 //!
 #![cfg(windows)]
-#![feature(ptr_internals, custom_attribute)]
 
 pub mod connection;
 pub mod datetime;

--- a/src/query.rs
+++ b/src/query.rs
@@ -6,7 +6,7 @@ use log::trace;
 use serde::de;
 use std::collections::HashMap;
 use std::ptr;
-use std::ptr::Unique;
+use std::ptr::NonNull;
 use widestring::WideCString;
 use winapi::um::wbemcli::IWbemClassObject;
 use winapi::{
@@ -289,7 +289,7 @@ impl WMIConnection {
             ))?;
         }
 
-        let pcls_wrapper = IWbemClassWrapper::new(Unique::new(pcls_obj));
+        let pcls_wrapper = IWbemClassWrapper::new(NonNull::new(pcls_obj));
 
         Ok(pcls_wrapper)
     }

--- a/src/result_enumerator.rs
+++ b/src/result_enumerator.rs
@@ -7,7 +7,7 @@ use failure::Error;
 use log::trace;
 use serde::de;
 use std::convert::TryInto;
-use std::{mem, ptr, ptr::Unique};
+use std::{mem, ptr, ptr::NonNull};
 use widestring::WideCString;
 use winapi::um::oaidl::VARIANT;
 use winapi::um::oleauto::VariantClear;
@@ -28,11 +28,11 @@ use winapi::{
 ///
 #[derive(Debug)]
 pub struct IWbemClassWrapper {
-    pub inner: Option<Unique<IWbemClassObject>>,
+    pub inner: Option<NonNull<IWbemClassObject>>,
 }
 
 impl IWbemClassWrapper {
-    pub fn new(ptr: Option<Unique<IWbemClassObject>>) -> Self {
+    pub fn new(ptr: Option<NonNull<IWbemClassObject>>) -> Self {
         Self { inner: ptr }
     }
 
@@ -114,14 +114,14 @@ impl Drop for IWbemClassWrapper {
 
 pub struct QueryResultEnumerator<'a> {
     wmi_con: &'a WMIConnection,
-    p_enumerator: Option<Unique<IEnumWbemClassObject>>,
+    p_enumerator: Option<NonNull<IEnumWbemClassObject>>,
 }
 
 impl<'a> QueryResultEnumerator<'a> {
     pub fn new(wmi_con: &'a WMIConnection, p_enumerator: *mut IEnumWbemClassObject) -> Self {
         Self {
             wmi_con,
-            p_enumerator: Unique::new(p_enumerator),
+            p_enumerator: NonNull::new(p_enumerator),
         }
     }
 }
@@ -172,7 +172,7 @@ impl<'a> Iterator for QueryResultEnumerator<'a> {
             pcls_obj
         );
 
-        let pcls_wrapper = IWbemClassWrapper::new(Unique::new(pcls_obj));
+        let pcls_wrapper = IWbemClassWrapper::new(NonNull::new(pcls_obj));
 
         Some(Ok(pcls_wrapper))
     }

